Add uploader to LegendasDivx subtitle results

redglory opened the pull request.

morpheus65535 commented on libs/subliminal_patch/providers/legendasdivx.py:
```diff
@@ -204,14 +202,20 @@ def _process_page(self, video, bsoup, querytext, videoname):
 
+            # get subtitle uploader
+            sub_header = _subbox.find("div", {"class" :"sub_header"}) 
+            uploader = sub_header.find("a").text if sub_header else '<n/a>'
```
else 'anonymous'

morpheus65535 replied:
Please make this change and I'll be able to merge your PR. Good work BTW!

morpheus65535 commented:
Only one small comment but the rest seems pretty good. Thanks!

redglory commented:
I've fixed this in 8b04941a3f71f49b661205f687c005a0b5f96d08

And I've fixed some things on the previous pull request 2dcfc433c3cfddb75a8c9678c60b2814feb78dfb

- now it searches for next pages if present
- fixed some exceptions which don't make sense  (fixes issue #962) and refined others
- improved some log debug messages

morpheus65535 commented:
> I've fixed this in [8b04941](https://github.com/morpheus65535/bazarr/commit/8b04941a3f71f49b661205f687c005a0b5f96d08)
> 
> And I've fixed some things on the previous pull request [2dcfc43](https://github.com/morpheus65535/bazarr/commit/2dcfc433c3cfddb75a8c9678c60b2814feb78dfb)
> 
> * now it searches for next pages if present
> * fixed some exceptions which don't make sense  (fixes issue #962) and refined others
> * improved some log debug messages

Thanks for doing this!
